app/components/views/TransactionsPage/SendTab/index.js

```diff
@@ -225,6 +228,15 @@ class Send extends React.Component {
     publishUnminedTransactions && publishUnminedTransactions();
   }
 
+  onKeyDown(e) {
+    if (e.keyCode === 13 && this.getIsValid()) {
+      var b = document.getElementsByClassName("content-send");
```

Comment:
This breaks react's patterns of not directly using dom references. You need to somehow propagate the enter event up the stack until it reaches the proper place.

The recently merged PR #1540 changed how transactions are signed and published to account for future trezor integration, so take a look at that (specifically at the new SendTransactionButton). You need to be able to propagate the event of the enter key up to that button using react idioms.

Reply:
fixed